### PR TITLE
Stylistic Loggin guide tweaks

### PR DESCRIPTION
**Pull request by MichalMaler**
Proper heading for the Example section and some additional styling fixes.

### CODE REVIEW

**Review by Ladicek — APPROVED** [review]

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -741,7 +741,7 @@ implementation("org.jboss.slf4j:slf4j-jboss-logmanager")
 
 === Use MDC to add contextual log information
 
-Quarkus overrides log MDC (Mapped Diagnostic Context) to improve the compatibility with its reactive core.
+Quarkus overrides log Mapped Diagnostic Context (MDC) to improve the compatibility with its reactive core.
```
Comment (jmartisk) [inline docs/src/main/asciidoc/logging.adoc:744]: "Quarkus overrides log Mapped Diagnostic Context" sounds quite odd to me

Could we say

"Quarkus overrides the logging Mapped Diagnostic Context" instead?

Comment (Ladicek) [reply docs/src/main/asciidoc/logging.adoc:744]: Agree, that sounds better.

**Review by michelle-purcell — COMMENTED** [review]
LGTM :+1:  While you are in there, can you fix line 336: Change `process and store` to ` processes and stores`. Thanks.

File: docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc]: Done and done.

**Review by Ladicek — APPROVED** [review]

**Pull request merged**